symbulate/results.py

```diff
@@ -286,22 +287,46 @@ class RVResults(Results):
 
     def plot(self, type=None, alpha=None, normalize=True, jitter=False, 
         bins=30, **kwargs):
+        if type is None:
+            pass
+        elif isinstance(type, str):
+            type = (type,)
+        elif not isinstance(type, (tuple, list)):
+            raise Exception("I don't know how to plot a " + str(type))
+            
         
         dim = get_dimension(self)
         if dim == 1:
             counts = self._get_counts()
             if type is None:
                 heights = counts.values()
                 if sum([(i > 1) for i in heights]) > .8 * len(heights):
```

Comment:
The logic in this line (to determine whether to treat the distribution as discrete or continuous) now appears multiple times within this file. You should factor it out as a function and put it in plot.py.